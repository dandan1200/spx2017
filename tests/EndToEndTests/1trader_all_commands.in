BUY 0 GPU 2 20;
SELL 1 Router 4 20;
AMEND 0 3 20;
CANCEL 1;
