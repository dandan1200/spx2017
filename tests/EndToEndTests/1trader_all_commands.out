MARKET OPEN;
ACCEPTED 0;
ACCEPTED 1;
AMENDED 0;
CANCELLED 1;
